src/main/resources/static/app/lab-request/lab-request-contents.html

```diff
@@ -224,6 +224,14 @@ <h4 class="h4">{{'Request details' | translate}}</h4>
         </p>
     </div>
 
+    <!-- Germline mutation -->
+    <div class="row">
+        <label class="control-label col-xs-4 col-md-2">{{ 'Germline mutation analysis' | translate }}</label>
+        <p class="col-xs-8 col-md-10">
+            {{labRequest.request.germlineMutation}}
```

Comment:
Shouldn't this also be `{{labRequest.request.germlineMutation ? 'Yes' : 'No' | translate}}`?